internal/microsoft_edge.py

```diff
@@ -199,6 +198,10 @@ def stop(self, job, task):
                     os.remove(self.wpt_etw_done)
                 except Exception:
                     pass
+        import _winreg
```

Comment:
Any chance of this throwing an exception?  The setup is inside of a try/except block.

Reply:
I guess better safe then sorry and put it into on. Oriented myself on the win32inet block on the edge.py code.